packages/ui/src/components/RadioGroup/radiogroup.tsx

```diff
@@ -0,0 +1,77 @@
+import { Radio as DefaultRadio } from 'antd'
+import classNames from 'classnames'
+import React, { useState } from 'react'
+
+import { Radio, Typography, Space } from '@open-condo/ui/src'
+import type { RadioProps } from '@open-condo/ui/src'
+
+import type { RadioGroupProps as DefaultRadioGroupProps } from 'antd'
+
+const RADIO_GROUP_CLASS_PREFIX = 'condo-radio'
+
+type RadioGroupOptionType = Pick<RadioProps, 'value' | 'label'> & {
+    radioProps?: Partial<Pick<RadioProps, 'labelProps' | 'disabled'>>
+}
+
+type RadioGroupType = {
+    name: string
+    options: Array<RadioGroupOptionType>
+}
+
+export type RadioGroupProps = {
+    icon: React.ReactElement
+    groups: Array<RadioGroupType>
+} & Pick<DefaultRadioGroupProps, 'value' | 'onChange' | 'disabled'>
+
+const GroupWithIcon: React.FC<RadioGroupType> = ({ name, options, children }) => {
```

Comment:
renaming children to icon `children: icon` or even using icon as separate prop will be more clear to read